tests/playwright/package.json

```diff
@@ -10,7 +10,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@playwright/test": "^1.42.1",
-    "@podman-desktop/tests-playwright": "^1.9.1",
+    "@podman-desktop/tests-playwright": "^0.0.202404081045-e2142a7",
```

Comment:
why do we need to use a non-tagged version there ?

Reply:
I'm not positive about 1.9.1 specifically, however we make changes to the test framework all the time so tagged version very quickly become out of data from that perspective.

Reply:
but then if you always want to grab the current "snapshot" of Podman Desktop, probably you should update the version during the e2e test else for example in a couple of days, it'll the same issue

Either we stick to the version being supported or if you need always the latest 'bits' then the test should take care of switching the dependency to the current published version of the test framework

or maybe two run are interesting.
Test against the latest stable version of Podman Desktop and test against the current development version of Podman Desktop

Reply:
After some advice from @odockal I've changed the npm package version for the test framework to have value `next`, this will ensure that the latest version of the test framework is used when running the tests and removes the need to manually update the version in the package.json file.

Reply:
so it won't do anything to have next

it's b/c we're pinning down versions into a yarn.lock file so versions are resolved only once (when we submit the yarn.lock file)

Reply:
ah yes, fair point, in that case it doesn't do anything.

Reply:
Unless we would remove the `@podman-desktop/tests-playwright` from the lock file, or we remove the file on PR check to make sure we always grab latest version.

Reply:
in PR check yaml, do `yarn add -D @podman-desktop/tests-playwright@next` in tests/playwright folder as first step

Reply:
done, added it before the lockfile is ran.